index.html

```diff
@@ -1,10 +1,34 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/main.css">
+    <title>Document</title>
 </head>
 <body>
 
+<header class="header" id="header-anchor">
+
+    <nav>
+        <div class="header__left-side">
+            <img src="images/Logotype.png" alt="" class="header__logotype__image">
+        </div>
+        <div class="header__right-side">
+            <ul class="nav-menu">
+                <li class="nav-elem"><a href="" class="nav-link">Home</a></li>
```

Comment:
I think better is when you have something like this:
`<li><a href="">Home</a></li>`

And your `style.css` something like this:
`.navmenu ul li { } `